provisioning/service/samples/createCertificate.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
```

Comment:
Please add the standard Microsoft copyright/license header is missing:
// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.

Reply:
Added